src/components/Button/index.js

```diff
@@ -3,6 +3,85 @@ import cn from 'classnames'
 
 import { string, func, bool, node, oneOfType, arrayOf } from 'prop-types'
 
+const getLoader = () => (
```

Comment:
Could you move this to another component and place it in `/src/components/`?
Maybe you can name it as `ButtonLoader`.